features/page_element_interaction.feature

```diff
@@ -32,17 +32,17 @@ Feature: Page element interaction
 
   Scenario: Wait for element
     When I navigate to the home page
```

Comment:
So this feature is interesting because 16 of 18 scenarios use `When I navigate to the home page`, so at least visually you'd see some benefit from a Background step here.

But then there are those 2 scenarios that would have to load a page after the home page is loaded. In this particular suite we're loading small, simple html pages so I don't think there'd be a performance hit. In the real world though, you really wouldn't want to double-load. And there are folks that are going to use this suite as an example of what to do in their own work.

Ultimately I'd probably add the Background step here for ease of maintenance.

Reply:
Both this one and the other (plural) feature need splitting out better. It feels (And the specs do as well), like a proper melting pot. But this PR already had a sizeable amount of refactors and I didn't want to overload the PR.

Are you happy for me to tackle this and the other one (plural), in a separate PR, where we look to split them out into 2/3 different features each?

Reply:
That seems reasonable, yes.